core/module.h

```diff
@@ -72,16 +60,15 @@ struct PbCommand {
   int mt_safe;
```

Comment:
Just wanted to confirm, the change seems to be introduced in snctl.cc (https://github.com/NetSys/bess/blame/d9070a8a6aea1be54f0f38a75a902cc4416c6c98/core/snctl.cc#L1295). Should I refactor RunCommand()?